cdap-ui/app/services/logsApi/my-logs-api.js

```diff
@@ -27,11 +27,18 @@ angular.module(PKG.name + '.services')
       appId: '@appId',
       programType: '@programType',
       programId: '@flowId',
-      runId: '@runId'
+      runId: '@runId',
+      start: '@start',
+      stop: '@stop'
     },
     {
+      getLogs: myHelpers.getConfig('GET', 'REQUEST', basepath + '/runs/:runId/logs', true),
+      getLogsJson: myHelpers.getConfig('GET', 'REQUEST', basepath + '/runs/:runId/logs?format=json', true),
+      getLogsStart: myHelpers.getConfig('GET', 'REQUEST', basepath + '/runs/:runId/logs?format=json?start=start', true),
```

Comment:
I'm not sure that API is correct. start=<epoch time>. If you are using start=start, it is sending the string start. Does the backend support that?